app/assets/javascripts/csv-upload/csv-upload.js

```diff
@@ -0,0 +1,9 @@
+window.onload = () => {
+  document.getElementById('upload-button').onclick = buttonClicked
+}
+
+function buttonClicked () {
+  document.getElementById('upload-button').disabled = true
+  document.getElementById('loading-message').innerText = 'Processing...'
+  window.setTimeout(() => window.location.assign(`${window.location.origin}/declare-partnership/multiple/failed`), 5000)
```

Comment:
So we will always fail this? Just making sure that's what UR wants to test :)

Reply:
Yep, that's right. That's what Dom asked me to build